cdn/bs3-cdn-css.sublime-snippet

```diff
@@ -1,7 +1,7 @@
 <snippet>
 	<content><![CDATA[
 <!-- Latest compiled and minified CSS -->
-<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/${1:3.3.4}/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/${1:3.3.5}/css/bootstrap.min.css">
```

Comment:
fiz a mesma atualizacao for maxcdn and for jquery for "http://" => jquery,min.js tambem